README.md

```diff
@@ -6,14 +6,14 @@ Repository for out-of-tree scheduler plugins based on scheduler framework.
 
 Starting with release v0.18.9 container images are available in the official k8s container registry
```

Comment:
Although v0.19.8 is not officially available yet, should we temporarily replace v0.18.9 with v0.19.8 here?

Reply:
I went with the suggestion from @denkensk to just put all of the versions and architectures into the table down below. This way we do not need to updated this paragraph for each release.